Tests/Controller/CRUDControllerTest.php

```diff
@@ -38,6 +38,9 @@
 /**
  * Test for CRUDController.
  *
+ * NEXT_MAJOR: Remove this legacy group.
```

Comment:
https://github.com/sonata-project/dev-kit/blob/master/project/CONTRIBUTING.md#the-base-branch